cmd/promtool/main.go

```diff
@@ -777,6 +777,10 @@ func checkRules(files []string, ls lintConfig) (bool, bool) {
 		rgs, errs := rulefmt.ParseFile(f)
 		if errs != nil {
 			failed = true
+			for _, e := range errs {
+				fmt.Fprintln(os.Stderr, e.Error())
+				hasErrors = hasErrors || !errors.Is(e, lintError)
+			}
 			continue
```

Comment:
Should we only continue if there are hard errors? (not continue if there is only lint issues)

Reply:
Aha, you are right.